DQM/Physics/python/DQMPhysics_cff.py

```diff
@@ -33,6 +34,7 @@
                            *HiggsDQM
                            *ExoticaDQM
                            *B2GDQM
+                           *CentralitypADQM
```

Comment:
Yes, this requires products "pACentrality" from pA era.  But I have tested successfully this using the workflow 1333 which does not have product "pACentrality".